crates/hyperswitch_connectors/src/connectors/nexixpay/transformers.rs

```diff
@@ -142,87 +552,278 @@ impl<F, T> TryFrom<ResponseRouterData<F, NexixpayPaymentsResponse, T, PaymentsRe
+        let payment_method_data: PaymentMethodData =
+            item.router_data.request.payment_method_data.clone().ok_or(
+                errors::ConnectorError::MissingRequiredField {
+                    field_name: "payment_method_data",
+                },
+            )?;
+        let operation_id = item
+            .router_data
+            .request
+            .connector_transaction_id
+            .clone()
+            .ok_or(errors::ConnectorError::MissingRequiredField {
+                field_name: "connector_transaction_id",
+            })?;
+        let capture_type = get_nexixpay_capture_type(item.router_data.request.capture_method)?;
+
+        let order_id = item.router_data.connector_request_reference_id.clone();
+        let amount = item.amount.clone();
+        let billing_address = Address {
+            name: item.router_data.get_optional_billing_full_name().clone(),
+            street: item.router_data.get_optional_billing_line1(),
+            additional_info: None,
+            city: item.router_data.get_optional_billing_city(),
+            post_code: item.router_data.get_optional_billing_zip(),
+            province: None,
+            country: item.router_data.get_optional_billing_country(),
+        };
+        let customer_info = CustomerInfo {
+            card_holder_name: None,
+            billing_address: billing_address.clone(),
+            shipping_address: billing_address,
+        };
+        let order_data = Order {
+            order_id,
+            amount,
+            currency: item.router_data.request.currency,
+            description: None,
+            custom_field: None,
+            customer_info: Some(customer_info),
+        };
+        let connector_metadata = item.router_data.request.connector_meta.clone().ok_or(
+            errors::ConnectorError::MissingRequiredField {
+                field_name: "connector_meta",
+            },
+        )?;
+        let three_d_s_complete_auth_request_data =
+            serde_json::from_value::<ThreeDSCompleteAuthRequestData>(connector_metadata)
+                .change_context(errors::ConnectorError::ParsingFailed)?;
+        let three_d_s_auth_data = ThreeDSAuthData {
+            three_d_s_auth_response: three_d_s_complete_auth_request_data.three_d_s_auth_response,
+            authentication_value: three_d_s_complete_auth_request_data
+                .three_d_s_auth_result
+                .authentication_value,
+        };
+        let card: Result<NexixpayCard, error_stack::Report<errors::ConnectorError>> =
+            match payment_method_data {
+                PaymentMethodData::Card(req_card) => Ok(NexixpayCard {
+                    pan: req_card.clone().card_number,
+                    expiry_date: req_card.clone().get_expiry_date_as_mmyy()?,
+                    cvv: None,
+                }),
+                _ => Err(
+                    errors::ConnectorError::NotImplemented("Payment methods".to_string()).into(),
```

Comment:
avoid using defaults
```suggestion
                    errors::ConnectorError::NotImplemented("Payment method".to_string()).into(),
```